src/gretel_synthetics/train.py

```diff
@@ -59,9 +60,13 @@ def annotate_training_data(store: BaseConfig):
     # required for sentencepiece to tokenize newline characters
     logging.info(f"Annotating training data from {store.input_data}")
     training_text = []
-    with open(store.input_data, 'r', encoding='utf-8') as infile:
+    with open(store.input_data, 'r', encoding='utf-8', errors='replace') as infile:
         for line in infile:
-            training_text.append(f"{line.strip()}")
+            if (store.max_lines == 0) or (len(training_text) < store.max_lines):
```

Comment:
nit: I prefer moving the `break` case up so as to minimize indentation of the happy-path code for better readability, i.e.,
```
if store.max_lines and len(training_text) >= store.max_lines:
  break
line = line.strip()....
```
(feel free to ignore)

Reply:
that's definitely more clean- updated